README.md

```diff
@@ -5,40 +5,154 @@ Please edit this README.md file with your answers to the worksheet questions.
 ## Question 1
 
 ### a
+A | B | C | A AND B AND NOT C
+-------------|-------------|-------------|-------------
+False	|	False	|	False	|	False
+False	|	False	|	True	|	False
+False	|	True	|	False	|	False
+False	|	True	|	True	|	False
+True	|	False	|	False	|	False
+True	|	False	|	True	|	False
+True	|	True	|	False	|	True
+True	|	True	|	True	|	False
 
 ### b
+A| B | C | A AND NOT (B AND NOT C)|
+-------------|-------------|-------------|-------------
+False	|	False	|	False	|	False
+False	|	False	|	True	|	False
+False	|	True	|	False	|	False
+False	|	True	|	True	|	False
+True	|	False	|	False	|	True
+True	|	False	|	True	|	True
+True	|	True	|	False	|	False
+True	|	True	|	True	|	True
 
 ### c
+A | B | C | (A OR NOT B) AND (A OR C)
+-------------|-------------|-------------|-------------
+False	|	False	|	False	|	False
+False	|	False	|	True	|	True
+False	|	True	|	False	|	False
+False	|	True	|	True	|	False
+True	|	False	|	False	|	True
+True	|	False	|	True	|	True
+True	|	True	|	False	|	True
+True	|	True	|	True	|	True
 
 ### d
+A | B | C| D |A AND NOT (B OR NOT C) AND (NOT A AND D)
+-------------|-------------|-------------|-------------|-------------
+False	|	False	|	False	|	False	|	False
+False	|	False	|	False	|	True	|	False
+False	|	False	|	True	|	False	|	False
+False	|	False	|	True	|	True	|	False
+False	|	True	|	False	|	False	|	False
+False	|	True	|	False	|	True	|	False
+False	|	True	|	True	|	False	|	False
+False	|	True	|	True	|	True	|	False
+True	|	False	|	False	|	False	|	False
+True	|	False	|	False	|	True	|	False
+True	|	False	|	True	|	False	|	False
+True	|	False	|	True	|	True	|	False
+True	|	True	|	False	|	False	|	False
+True	|	True	|	False	|	True	|	False
+True	|	True	|	True	|	False	|	False
+True	|	True	|	True	|	True	|	False
 
-## Question 2
-
-### a
-
-### b
 
-### c
+## Question 2
 
-### d
+![Question_2_Answers](https://i.imgur.com/YvaYtgY.png)
```

Comment:
Part d: you have `B or not C` instead of `not (B or not C)` -- need to make it a `nor` gate or add a `not` to the output. 

Otherwise all correct.